src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.spec.ts

```diff
@@ -103,6 +103,24 @@ describe('LocalGameWrapperComponent', () => {
         await testUtils.expectMoveSuccess('#click_3', P4Move.THREE);
         testUtils.expectElementToExist('#draw');
     }));
+    it('should not allow clicks after the end of the game', fakeAsync(async() => {
+        // Given an ended game
+        const board: PlayerOrNone[][] = [
+            [_, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _],
+            [X, _, _, _, _, _, _],
+            [X, O, _, _, _, _, _],
+            [X, O, O, O, _, _, _],
+        ];
+        const state: P4State = new P4State(board, 41);
+        testUtils.setupState(state);
+        await testUtils.expectMoveSuccess('#click_0', P4Move.ZERO);
```

Comment:
somme toute in-mgp-we-trust, on peut direct créer un state fini, non ?

Reply:
ben... non :D On peut créer un état en cours de partie, mais on peut pas créer un état fini et triggerer la logique de fin de partie visiblement. Donc : créer l'état avant la fin, puis faire le dernier mouvement permet de régler ça.

Reply:
Du coup c'est
Given a board about to finish
When doing the finish move [and trigerring what's up and what not]
THEN clicking again should fail ?
non ?

Reply:
yep, reformulé !